src/deployment/4_deployL1Applications.js

```diff
@@ -0,0 +1,149 @@
+/* eslint-disable no-await-in-loop, no-use-before-define, no-lonely-if, import/no-dynamic-require, global-require */
+/* eslint-disable no-console, no-inner-declarations, no-undef, import/no-unresolved, no-restricted-syntax */
+const path = require('path');
+const fs = require('fs');
+const { expect } = require('chai');
+const { ethers, upgrades } = require('hardhat');
+require('dotenv').config({ path: path.resolve(__dirname, '../.env') });
+
+const pathOutputJson = path.join(__dirname, './deploy_output_l1_applications.json');
+const pathOngoingDeploymentJson = path.join(__dirname, './deploy_ongoing_l1_applications.json');
+
+// const deployParameters = require('./deploy_parameters.json');
+const deployParameters = {};
+
+const delay = ms => new Promise(res => setTimeout(res, ms));
+
+async function main() {
+
+    // Check if there's an ongoing deployment
+    let ongoingDeployment = {};
+    if (fs.existsSync(pathOngoingDeploymentJson)) {
+        ongoingDeployment = require(pathOngoingDeploymentJson);
+    }
+
+    /*
+     * Check deploy parameters
+     * Check that every necessary parameter is fullfilled
+     */
+    /*
+    const mandatoryDeploymentParameters = [
```

Comment:
If this is empty, we can remove it?

Reply:
Let's leave an empty list then refactor it out

Reply:
lemme combine it with something general when I refactor so they all work the same way and if there's nothing you have to specify, we say that.